packages/react/src/patterns/blocks/LeadSpaceBlock/__stories__/LeadSpaceBlock.stories.js

```diff
@@ -0,0 +1,120 @@
+import { object, select, withKnobs, text } from '@storybook/addon-knobs';
+import { HorizontalRule } from '../../../../components/HorizontalRule';
+import LeadSpaceBlock from '../LeadSpaceBlock.js';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Blocks)|LeadSpaceBlock', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const title = 'Continuous delivery';
+
+    const heading = text(
+      'Heading (required)',
+      'Innovate like a startup and scale for the enterprise '
+    );
+
+    const copy = `Automate your software release process with continuous delivery (CD)—the most 
+    critical part of adopting DevOps. Build, test, and deploy code changes quickly, 
+    ensuring software is always ready for deployment.`;
+
+    const mediaType = select(
+      'mediaType (optional)',
+      ['image', 'video', 'none'],
+      'image'
+    );
+
+    const video = {
+      videoId: '0_uka1msg4',
+      showCaption: true,
+    };
+
+    const image = {
+      heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+      image: {
+        sources: [
+          {
+            src: 'https://dummyimage.com/320x180/ee5396/161616&text=16:9',
+            breakpoint: 320,
+          },
+          {
+            src: 'https://dummyimage.com/400x225/ee5396/161616&text=16:9',
+            breakpoint: 400,
+          },
+          {
+            src: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
+            breakpoint: 672,
+          },
+        ],
+        alt: 'Image alt text',
+        defaultSrc: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
+      },
+    };
+
+    const mediaData = mediaType === 'image' ? image : video;
+
+    const linkListProps = {
+      heading: text('link list heading:', 'Featured products'),
+      items: object('link list items array', [
+        {
+          type: 'local',
+          copy: 'IBM Cloud Continuous Delivery',
+          cta: {
+            href: 'https://ibm.com',
+          },
+        },
+        {
+          type: 'local',
+          copy: 'UrbanCode',
+          cta: {
+            href: 'https://ibm.com',
+          },
+        },
+        {
+          type: 'download',
+          copy: 'View all products',
+          cta: {
+            href: 'https://ibm.com',
+          },
+        },
+      ]),
+    };
+
+    const types = ['local', 'external'];
+    const ctaProps = {
+      style: 'button',
+      type: 'local',
+      buttons: [
+        {
+          type: select('ContentBlock | CTA type', types, types[0]),
+          copy: text('ContentBlock | CTA copy', 'Contact sales'),
+          href: 'https://example.com/',
+        },
+      ],
+    };
+
+    return (
+      <div className="bx--grid">
+        <div className="bx--row">
+          <div className="bx--col-sm-4 bx--col-lg-8">
```

Comment:
can you add `bx--offset-lg-4` to the className to center it in the story?

Reply:
@annawen1 As per the sketch it is left aligned, the content occupies the 8 column and the horizontal rule occupies the 12 col. Do we still need this in the center?

@jeffchew

Reply:
@greeshmareddyg We center in storybook only to give some uniformity to the layout, and to show how it would live in the grid. Otherwise you're correct, it would be left-aligned in its parent container.

Reply:
@annawen1 @kennylam aligned the content to center.